OpenSim/Simulation/Test/testDelay.cpp

```diff
@@ -603,14 +632,19 @@ void testDiscontinuousInput() {
     // The "step" output switches at 0.5 seconds, but the delay is 0.2 seconds,
     // so the delay output should still be 1.5.
     //std::cout << comp->stepDelay.getValue(ts.getState()) << " " << std::endl;
-    SimTK_TEST_EQ(comp->stepDelay.getValue(ts.getState()), 1.5);
-    //std::cout << comp->discontDelay.getValue(ts.getState()) << " " << std::endl;
-    SimTK_TEST_EQ(comp->discontDelay.getValue(ts.getState()), 0.46);
 
+    const Delay& stepDelay = comp->getComponent<Delay>("/discontinuousInputTesting/stepDelay");
+    const Delay& discontDelay = comp->getComponent<Delay>("/discontinuousInputTesting/discontDelay");
```

Comment:
The current way to fish out a member subcomponent is shown [here](https://github.com/opensim-org/opensim-core/blob/de24648dbc6c1342d74bf7ba8b792fd4c08edbc6/OpenSim/Actuators/Thelen2003Muscle.cpp#L103).